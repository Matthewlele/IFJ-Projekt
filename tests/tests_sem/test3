def bar ()
  x = 5
  while x > 0 do
    print x
    x = y - 1
  end
  x
end

x = 5

while x <= 5 do
  print x
  x = x +1
end
