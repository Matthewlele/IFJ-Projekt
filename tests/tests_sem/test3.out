expected SEMANTIC ERR: line 5
